nni/common/concrete_trace_utils/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
```

Comment:
Suggest adding module-level docstrings to explain what this is all about.

Reply:
ok